styles.scss

```diff
@@ -0,0 +1,299 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  background: $first-color;
+  font-family: coiny;
+}
+
+body {
+  .container {
+    margin: auto;
+    padding: 5px;
+    border: 3px solid $second-color;
+    border-radius: 10px;
+    width: 90%;
+    height: 95vh;
+    display: grid;
+    grid-template: 12% 79% 1fr / 100%;
+    grid-row-gap: 5px;
+    align-content: stretch;
+    align-items: stretch;
+
+    .header, .game-wrapper, .sentence-wrapper {
+      border: inherit;
+      border-radius: inherit;
+      background-color: $fourth-color;
+    }
+
+    .header {
+      .title {
+        text-align: center;
+        margin-top: .1em;
+        margin-bottom: 0;
+      }
+      .btn {
+        display: block;
+        margin: 0 auto .5em auto;
+        width: 10em;
+        font-family: inherit;
+        border: .2em solid $first-color;
+        border-radius: 1em;
+        color: $first-color;
+        background-color: $second-color;
+        &:hover {
+          cursor: pointer;
+          background-color: $third-color;
+          color: $second-color;
+          border: .25em solid $second-color;
+          margin: 0 auto .4em auto;
+        }
+      }
+    }
+
+    .game-wrapper {
+      display: grid;
+      grid-template: repeat(8, 12%) / repeat(7, 14%);
+      justify-content: center;
+      align-content: center;
+      grid-gap: .2%;
+
+      @media screen and (max-width: 600px) {
```

Comment:
try to create mixin for media screens ;) look in docs what is `@content`. 

Also, you could try `mobile first` approach. So - at first write version for mobile, then use `min-width` and overwrite attributes. You are doing in the opposite way right now ;) 

Also - read about mobile first during making a poo 💩